packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/template/DescriptionFieldTemplate.tsx

```diff
@@ -0,0 +1,18 @@
+import { DescriptionFieldProps } from '@rjsf/utils'
+import React from 'react'
+import { Typography } from '@mui/material'
+import FieldDescriptionTable from './FieldDescriptionTable'
+
+export default function DescriptionFieldTemplate(props: DescriptionFieldProps) {
+  const { description, schema = {} } = props
+
+  if (schema.type === 'object') {
+    return (
+      <Typography variant={'body1'} sx={{ my: 2 }}>
```

Comment:
What does this my:2 (margin-top/margin-bottom) result in today?  (theme spacing x2?)

Reply:
Yep, since our spacing is 8px this is equivalent to `margin-top: 16px; margin-bottom: 16px;`